src/status_im/contexts/wallet/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -0,0 +1,121 @@
+  [rn/view {:style (style/buttons-inner-container margin-bottom)}
+   [button
+    (merge params
+           {:word         (first options)
+            :margin-right 12})]
+   [button (merge params {:word (second options)})]])
+
+(defn- view-internal
+  []
+  (let [random-indices  (random-selection)
+        quiz-index      (reagent/atom 0)
+        incorrect-count (reagent/atom 0)
+        show-error?     (reagent/atom false)]
+    (fn []
+      (let [current-word-index (get random-indices (min @quiz-index 3))
+            current-word       (get temp/secret-phrase current-word-index)
+            options            (random-words-with-string temp/random-words current-word)
+            button-params      {:quiz-index      quiz-index
+                                :current-word    current-word
+                                :incorrect-count incorrect-count
+                                :show-error?     show-error?}]
+        [rn/view {:style {:flex 1}}
+         [quo/page-nav
+          {:icon-name           :i/arrow-left
+           :on-press            #(rf/dispatch [:navigate-back])
+           :accessibility-label :top-bar}]
+         [quo/text-combinations
+          {:container-style style/header-container
+           :title           (i18n/label :t/check-your-backup)
+           :description     (i18n/label :t/confirm-the-position)}]
+         [rn/flat-list
+          {:data                    random-indices
+           :render-fn               (fn [num index] [quo/quiz-item
+                                                     {:state    (if (= @quiz-index index)
+                                                                  (if (and (pos? @incorrect-count)
+                                                                           @show-error?)
+                                                                    :error
+                                                                    :empty)
+                                                                  (if (> @quiz-index index)
+                                                                    :success
+                                                                    :disabled))
+                                                      :word     (get temp/secret-phrase num)
+                                                      :number   (inc num)
+                                                      :on-press #(when (= @quiz-index index)
+                                                                   (reset! show-error? false))}])
+           :separator               [rn/view {:style {:height 8}}]
+           :content-container-style {:padding-horizontal 20}}]
+         [rn/view {:style style/buttons-container}
+          [buttons-row
+           (merge button-params
+                  {:margin-bottom 12
+                   :options       (subvec options 0 2)})]
+          [buttons-row (merge button-params {:options (subvec options 2 4)})]]]))))
```

Comment:
Same about this `merge`.

Since we only want to add some keys, `assoc` is enough

Reply:
Same ^